src/APICalls/APICalls.js

```diff
@@ -20,6 +29,17 @@ export const getAllCourses = () => {
     })
 }
 
+export const getAllEvents = () => {
+  return fetch(endpoints.eventsDev)
+    .then(resp => {
+      if (!resp.ok) {
+        throw new Error('Can\'t fetch any events, please try again!')
+      } else {
+        return resp.json()
+      }
+    })
```

Comment:
We still need to add a catch for these fetches

Reply:
nice _catch_